### DISCUSSION TITLE
read only mode

### DISCUSSION THREAD

**Discussion by WarlockFish** [outline/outline#4250] [Ideas]
After I have written the document, the next time I view it, the mouse and keyboard may be touched by mistake, resulting in the deletion of text. I hope that read-only mode and writing mode can be available

**Comment by Limezy** [comment]
There will be very soon a mode to disable "seamless editing". I think it's already shipped for the SaaS and will be included in the next self hosted release.

**Reply by Limezy** [reply]
https://github.com/outline/outline/pull/4203

**Comment by WarlockFish** [comment]
thank you 

i disable "seamless editing" , there is read-only mode, but i find  when writing text, it becomes full screen mode and the sidebar disappears.
![2022-10-10_17-11](https://user-images.githubusercontent.com/30708115/194833238-df49e028-580c-4dfd-932f-75cdbfac7ca5.png)


i hope the "seamless editing"‘'s page setup and the read only mode. i  I look forward to it in the next version

**Reply by Limezy** [reply]
I haven't try to build Outline with that merged PR yet.
I'll try and tell you if I got the same issue.
